testament/important_packages.nim

```diff
@@ -118,7 +118,7 @@ pkg "nitter", "nim c src/nitter.nim", "https://github.com/zedeus/nitter"
 pkg "norm", "testament r tests/sqlite/trows.nim"
 pkg "npeg", "nimble testarc"
 pkg "numericalnim", "nimble nimCI"
-pkg "optionsutils"
+pkg "optionsutils", allowFailure = true # pending https://github.com/PMunch/nim-optionsutils/pull/6
```

Comment:
```suggestion
pkg "optionsutils", "nim c --gc:arc src/optionsutils" # pending https://github.com/PMunch/nim-optionsutils/pull/6
```

Reply:
Because the test of weave works by pure chance (the dependency which it needs seems to be installed by other packages) 😆